README.md

```diff
@@ -1,3 +1,7 @@
 # @0x/quote-server
 
-An RFQ quote server that can be used to provide quotes via [0x API](https://0x.org/api).
+A reference implementation of the [Maker Endpoint API](https://0x.org/docs/guides/rfqt-in-the-0x-api#maker-endpoint-specification) that's used in the RFQ-T model within the [0x API](https://0x.org/api).  For more details, see [RFQ-T in the 0x API](https://0x.org/docs/guides/rfqt-in-the-0x-api).
```

Comment:
I think it'd be good to include example code of how to import the server and run it with a dummy quoter